src/psyclone/dynamo0p3.py

```diff
@@ -2024,13 +2025,16 @@ def _declare_maps_stub(self, parent):
         from psyclone.f2pygen import DeclGen
         api_config = Config.get().api_conf("dynamo0.3")
 
+        # Stubs are not connected to an InvokeSchedule and can use a dummy
```

Comment:
dummy -> their own?

Reply:
Changed this, and all similar comments in this file.